### DISCUSSION TITLE
Schedule for next release?

### DISCUSSION THREAD

**Discussion by isaacovercast** [sandialabs/toyplot#215] [General]
Hello. First, thanks for the awesome package! It's really useful and I appreciate all your work on it.

I see there was some recent work on supporting numpy 2, and I wonder if there's a plan for releasing the next toyplot version? The conda-forge package is still at 1.0.3, which looks like the most recent tag.

Thanks a lot!

**Comment by tshead2** [comment]
I've been remiss in not formally releasing this.  Will get it done ASAP.  In the meantime, you can pip install directly from the git repo if you need these fixes.

**Comment by tshead2** [comment]
Toyplot 2.0.0 has been released ... enjoy!
